ui/src/layouts/Footer/index.tsx

```diff
@@ -106,6 +115,17 @@ const FooterWrapper = styled.footer`
     }
   }
   .footer__bottom {
+    .flex {
```

Comment:
let's try to avoid putting styles on reserved utility classes, it might make more sense to give this a different class name and put the styles on there instead